src/validation/error.messages.ts

```diff
@@ -97,5 +97,7 @@ export enum ErrorMessages {
   OCCUPATION_INVALID_CHARACTERS = "Occupation must only include letters a to z, numbers, and special characters such as hyphens, spaces and apostrophes",
   ROLES_AND_RESPONSIBILITIES_INVALID_CHARACTERS = "Role and responsibilities must only include letters a to z, numbers, and special characters such as hyphens, spaces and apostrophes",
   CONTACT_NAME_INVALID_CHARACTERS = "Contact name must only include letters a to z, numbers, and special characters such as hyphens, spaces and apostrophes",
-  EMAIL_INVALID_FORMAT = "Enter an email address in the correct format, like name@example.com"
+  EMAIL_INVALID_FORMAT = "Enter an email address in the correct format, like name@example.com",
+  // Empty fields
+  TRUST_DATA_EMPTY = "Paste the trust information from the Excel document into the box."
```

Comment:
Empty field check texts are at the top of this file (although admittedly not commented, so not clear). Suggest moving this up to just under line 31 though, just to keep them all together.